components/checkbox/stories/checkbox.stories.js

```diff
@@ -84,7 +105,34 @@ export default {
 	},
 };
 
-export const Default = Template.bind({});
+const CheckboxGroup = ({
+	customStyles = {},
+	...args
+}) => {
+	return html`
+		<div style="display: flex; flex-direction: column; padding: 1rem">
+			${Template({
+				...args,
+				iconName: undefined,
+			})}
+			${Template({
+				...args,
+				isChecked: true,
+			})}
+			${Template({
+				...args,
+				isIndeterminate: true
+			})}
```

Comment:
Can we update this label to say "Checkbox Indeterminate" so that it's clear what the example is showing?